scripts/migration/turso-to-railway/02-transform.ts

```
@@ -0,0 +1,297 @@
+  }));
+
+  console.log(`\nTransform complete:`);
+  console.log(`  Pages: ${transformedPages.length}`);
+  console.log(
+    `  Page contents (Y.Doc): ${pageContents.length} (${ydocSuccessCount} full, ${errors.length} fallback/error)`,
+  );
+  console.log(`  Links: ${transformedLinks.length}`);
+
+  if (errors.length > 0) {
+    console.log(`\n  Conversion errors (${errors.length}):`);
+    errors.slice(0, 10).forEach((e) => console.log(`    - ${e.title}: ${e.error}`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)